tools/parsinginjs/src/main/java/org/enso/parsinginjs/Parsinginjs.java

```diff
@@ -0,0 +1,15 @@
+package org.enso.parsinginjs;
+
+import org.enso.compiler.core.EnsoParser;
+
+public class Parsinginjs {
```

Comment:
The _main_ entry point. Shows we can instantiate `EnsoParser` and `compile` some code. An `IR` tree is created.